libs/experimental/langchain_experimental/graph_transformers/llm.py

```diff
@@ -824,7 +825,9 @@ def __init__(
                 relationship_properties,
                 self._relationship_type,
             )
-            structured_llm = llm.with_structured_output(schema, include_raw=True)
+            structured_llm = llm.with_structured_output(
+                schema, method=method, include_raw=True
```

Comment:
Sorry I misunderstood - let's pass this in as a kwarg when it's defined, but let's remove the default of `json_schema` above.

Also open to something more general like `structured_output_kwargs: Optional[dict]` where you'd achieve this with `structured_output_kwargs={"method": "json_schema"}`

Reply:
Thanks for the review.  I went with `structured_output_kwargs` in the latest commit.